docs/howto/getting_started_markdown.md

```diff
@@ -0,0 +1,173 @@
+
+### Create Your Production Installer
+
+After you have implemented and tested your new Ayx Plugin Tool, you
+might want to distribute it to others. In order to do this, you must
+create a YXI installer. The Core SDK CLI provides a command for
+packaging your workspace and tools into an installer.
+
+Run the `create-yxi` command to generate a YXI installer that is ready
+for distribution.
+
+By default, the name of the YXI produced is based on the Ayx Plugin
+Workspace directory name, but you can override the default with the
+`--yxi-name` flag. Alternatively, you can modify the `yxi_name` field in
+the `ayx_workspace.json` file. (See the [Workspace
+Configurations](https://extensibility.pages.git.alteryx.com/ayx-sdks/python-sdk/getting_started.html#workspace-configurations)
+section for details).
+
+### Build the Ayx Plugin Tool
+
+The [sdk-workspace-init]{.title-ref} command sets up the folder
+structure with a `backends` folder, `ui` folder, and `configuration`
+folder. These folders have all the files necessary for Alteryx Designer
+to integrate with your plugin.
+
+Next we\'re going to build your new Ayx Plugin Tool into Designer, where
+you can drag it onto the canvas and connect it to other Designer tools.
+To build `MyFirstTool` into the Designer application, use the
+`designer-install` command.
+
+Use this command to build MyFirstTool into Designer:
+
+`ayx_plugin_cli designer-install`
+
+**Congratulations!**
+
+You just created your first Alteryx Plugin Tool! You can now open
+Alteryx Designer and find your new tool in the Tool Palette.
+
+### Ayx Plugin Tool Execution in Designer
+
+When Designer runs a tool, it must look for an engine to use. In the
+case of our MyFirstTool Python tool, the engine is itself the *Python
+interpreter*. The interpreter is built out of Anaconda and includes all
+of the packages indicated in `requirements.txt`.
+
+The ToolFamily defined in the Config.xml file within the Ayx Plugin
+Workspace defines the name of the virtual environment created for all of
+your Ayx Plugin tools. By default, a ToolFamily is set up for your Ayx
+Plugin Workspace for all of your tools to use.
+
+The YXI installation package includes all of the interpreter information
+so that the Python interpreter can be recreated on any machine when the
+tools are installed.
+
+The `EngineDLLEntryPoint` within the Config.xml file points to the file
+that contains a class definition that inherits from `Plugin`. This file
+can be changed at any time to any Python file as long as it contains a
+`Plugin` class that registers with the Alteryx Core SDK.
+
+-   For a detailed explanation of the Config.xml file, see [Tool
+    Configurations](https://extensibility.pages.git.alteryx.com/ayx-sdks/python-sdk/config_xml.html).
+-   For information on the `main.py` file within your Ayx Plugin Tool,
+    see [Plugin Code
+    Overview](https://extensibility.pages.git.alteryx.com/ayx-sdks/python-sdk/plugin_code.html).
```

Comment:
TW review:

For a detailed explanation of the Config.xml file, go to [Tool
    Configurations](https://extensibility.pages.git.alteryx.com/ayx-sdks/python-sdk/config_xml.html).
-   For information on the `main.py` file within your Ayx Plugin Tool,
    go to [Plugin Code
    Overview](https://extensibility.pages.git.alteryx.com/ayx-sdks/python-sdk/plugin_code.html).